src/main.rs

```diff
@@ -91,9 +91,19 @@ enum Command {
     New {
         /// The name of the project.
         name: String,
+
+        /// Whether or not to fetch the latest template online.
+        #[cfg_attr(feature = "fetch-template", arg(long, default_value = "true"))]
+        #[cfg_attr(not(feature = "fetch-template"), arg(skip = true))]
+        use_internet: bool,
     },
     /// Creates a new vexide project in the current directory
-    Init,
+    Init {
+        /// Whether or not to fetch the latest template online.
+        #[cfg_attr(feature = "fetch-template", arg(long, default_value = "true"))]
+        #[cfg_attr(not(feature = "fetch-template"), arg(skip = true))]
+        use_internet: bool,
```

Comment:
It might be worth considering separating this argument into its own struct and parsing it flattened because it's a significant amount of code duplication. Also I'm not entirely sure about its name.